src/config/config.ts

```diff
@@ -502,6 +502,15 @@ export class Configuration {
     };
   }
 
+  get p2b(): ExchangeConfig {
+    return {
+      apiKey: process.env.P2B_KEY,
+      secret: process.env.P2B_SECRET,
+      withdrawKeys: splitWithdrawKeys(process.env.P2B_WITHDRAW_KEYS),
```

Comment:
Update Bicep

Reply:
Hehe jap I have added it locally already 😁 
Are the parameters in config the right ones? Do we need the withdrawKeys for example?